src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/alerts/AlertsSynchronisationIntTest.kt

```diff
@@ -396,6 +396,112 @@ class AlertsSynchronisationIntTest : SqsIntegrationTestBase() {
       }
     }
 
+    @Nested
+    @DisplayName("When alert was created in NOMIS but on previous booking")
+    inner class NomisCreatedOnPreviousBooking {
+      private val bookingId = 12345L
+      private val alertSequence = 3L
+      private val offenderNo = "A3864DZ"
+
+      @BeforeEach
+      fun setUp() {
+        alertsMappingApiMockServer.stubGetByNomisId(status = NOT_FOUND)
+      }
+
+      @Nested
+      inner class WhenRelevantAlert {
+        private val dpsAlertId = "a04f7a8d-61aa-400c-9395-f4dc62f36ab0"
+
+        @BeforeEach
+        fun setUp() {
+          alertsNomisApiMockServer.stubGetAlert(
+            bookingId = bookingId,
+            alertSequence = alertSequence,
+            alert = alert(bookingId = bookingId, alertSequence = alertSequence).copy(
+              bookingSequence = 2,
+              isAlertFromPreviousBookingRelevant = true,
+              alertCode = CodeDescription("XNR", "Not For Release"),
+              type = CodeDescription("X", "Security"),
+              audit = alert().audit.copy(
+                auditModuleName = "OCDALERT",
+              ),
+            ),
+          )
+          dpsAlertsServer.stubPostAlert(dpsAlert().copy(alertUuid = UUID.fromString(dpsAlertId)))
+          alertsMappingApiMockServer.stubPostMapping()
+          awsSqsAlertOffenderEventsClient.sendMessage(
+            alertsQueueOffenderEventsUrl,
+            alertEvent(
+              eventType = "ALERT-INSERTED",
+              bookingId = bookingId,
+              alertSequence = alertSequence,
+              offenderNo = offenderNo,
+            ),
+          ).also { waitForAnyProcessingToComplete() }
+        }
+
+        @Test
+        fun `will create alert in DPS`() {
+          dpsAlertsServer.verify(
+            postRequestedFor(urlPathEqualTo("/alerts"))
+              .withRequestBody(matchingJsonPath("alertCode", equalTo("XNR"))),
+          )
+        }
+      }
+
+      @Nested
+      inner class WhenIrrelevantAlert {
+        @BeforeEach
+        fun setUp() {
+          alertsNomisApiMockServer.stubGetAlert(
+            bookingId = bookingId,
+            alertSequence = alertSequence,
+            alert = alert(bookingId = bookingId, alertSequence = alertSequence).copy(
+              bookingSequence = 2,
+              isAlertFromPreviousBookingRelevant = false,
```

Comment:
I had to read that a few times, would something like relevantToFutureBookings be clearer?